### ISSUE TITLE
Elementor v2.7.5 and Wordpress  5.3 disallow editing pages

### ISSUE THREAD

**Issue by casaout** [elementor/elementor#9585]
## Prerequisites
- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest stable version of Elementor.

## Description

Since the latest update of Wordpress (to 5.3) and Elementor (2.7.5), it is not possible to edit pages anymore using Elementor. It shows a text-only page (without CSS). First it says "Session expired" "Please log in again. The login page will open in a new tab. After logging in you can close it and return to this page." and then it tries to load the actual Elementor Editor, but fails by showing a text-list of all Elementor items:
![image](https://user-images.githubusercontent.com/5212692/68952500-43c01b00-07c0-11ea-8b46-2a9a2d541fee.png)

In the Wordpress.org forums I found many users with the same issues, which is why I think this might be a bug, and not something specific to my set-up.

## Steps to reproduce

This is some additional information and steps I tried to fix the issue:
* also disabled all plugins
* also re-installed Elementor (which got me from v2.7.4 to v2.7.5; wordpress didn’t show the update)
* tried to rollback Elementor under Tools, but the dropdown doesn’t let me pick an older version (seems to be an issue with v2.7.5)
* I cannot get the Elementor safe mode to run
* the plugins-list weirdly shows a message as if I tried to deactivate Elementor
* I think the issue might have appeared after the recent wordpress release (v5.3). before logging in, it asked me if it can update the database
* I actually rolled back 30 days on GoDaddy’s backup feature (files + database), but after the rollback, wordpress just wants to update the database again
* I know it worked well ~1 week ago
* hoster: godaddy hosted wordpress deluxe
* it works with the default 2020 theme, but not with the <a href="https://themeforest.net/item/stratus-app-saas-product-showcase/13674236?s_rank=1">Stratus theme (from Themovation)</a> (which we are updating and using since 1+ year with Elementor)

## Isolating the problem

- [x] This bug happens with only Elementor plugin active (and Elementor Pro).
- [ ] This bug happens with a default WordPress theme active.
- [x] I can reproduce this bug consistently using the steps above.

## Environment
System Environment: https://pastebin.com/XVUbi4GB

The log says 'elementorFrontend is not defined', maybe that's the issue?

**Comment by bainternet** [comment]
@casaout 

we can't replicate, 
Please contact our support at support @elementor.com for guidance in resolving this issue and mention that you were directed from this issue.

thanks.

**Comment by casaout** [comment]
thank you, will do that.

**Comment by ProInns** [comment]
Hello dear, i am facing the same problem, how can I know if this issue has been solved. Thank you

**Comment by casaout** [comment]
@ProInns someone posted a solution on the WordPress forum, but I am not sure yet if it works:
https://wordpress.org/support/topic/session-expired-when-editing-with-elementor/#post-12137436

**Comment by ProInns** [comment]
@casaout ,thank you, will check it.

**Comment by ProInns** [comment]
@casaout , Hello André this actually worked for me, the issue has been solved. Thank you for sharing.

**Comment by mattab** [comment]
we're experiencing the same issue, and we're not using Envato market, we have purchased the Pro version directly from elementor. we just sent a support request to the elementor pro team.

**Comment by arirangz** [comment]
Hello,
Any news on this issue? The link provided didn't help.
Thansk!

**Comment by snnx** [comment]
I had the issue for about 6 weeks. Just updated to:

Elementor 2.8.2
Kirki 3.0.25
Formidable forms 4.03.06
Wordpress 5.3.2
Purged cache in Cloudflare
Updated my theme from Themovation

And it now works.

**Comment by big-fudge** [comment]
I have tried all the steps listed above, nothing works. I don't use cloudflare so I even purched the cache from my provider. Still an empty Elementor page telling me to start a new session.

Yesterday everything worked fine, now the elementor broke. I did update the WP version to 5.3.2 though. But that shouldn't be the problem after reading all the solutions using that WP version.

Any ideas? Contacted support of my themotivation theme and elementor already, but want to try here as well.

thx folks.
